src/schema/feeds.ts

```diff
@@ -923,6 +938,22 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       }
       return feedResolverV1(source, args, ctx, info);
     },
+    feedPreview: (
```

Comment:
What differentiates this from `feedByConfig` now?

Reply:
configuration and params are different